mifosng-android/src/main/java/com/mifos/mifosxdroid/adapters/ShareChargesAdapter.java

```diff
@@ -0,0 +1,125 @@
+package com.mifos.mifosxdroid.adapters;
+
+import android.app.Activity;
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.mifos.mifosxdroid.R;
+import com.mifos.mifosxdroid.injection.ActivityContext;
+import com.mifos.objects.templates.clients.ChargeOptions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by mayankjindal on 21/08/17.
+ */
+
+public class ShareChargesAdapter extends
+        RecyclerView.Adapter<ShareChargesAdapter.ViewHolder> {
+
+    private Context c;
+    private List<ChargeOptions> chargeOptionList = new ArrayList<ChargeOptions>();
+
+    @Inject
+    public ShareChargesAdapter(@ActivityContext Context context) {
+        c = context;
+    }
+
+    public List<ChargeOptions> getChargeOptionList() {
```

Comment:
Why are we exposing this list with this public getter ?

Reply:
Removed.